1d8e4b3df42986c282a26724745afd59c1cf18fe
src/main/java/org/apache/platypus/server/grpc/LuceneServer.java
--- 
+++ 
@@ -598,7 +598,7 @@
         public void status(HealthCheckRequest request, StreamObserver<HealthCheckResponse> responseObserver) {
             try {
                 HealthCheckResponse reply = HealthCheckResponse.newBuilder().setHealth(TransferStatusCode.Done).build();
-                logger.info("StopIndexHandler returned " + reply.toString());
+                logger.info("HealthCheckResponse returned " + reply.toString());
                 responseObserver.onNext(reply);
                 responseObserver.onCompleted();
             } catch (Exception e) {